library/src/main/java/com/squareup/timessquare/CalendarPickerView.java

```diff
@@ -621,6 +624,22 @@ public boolean selectDate(Date date, boolean smoothScroll) {
     return wasSelected;
   }
 
+  /**
+   * Use {@link DateUtils} to format the dates. If the version is lower than API 9, we will format
+   * using the old way, that means, SimpleDateFormat.
+   *
+   * @see DateUtils
+   */
+  private String formatMonthDate(Date date, Locale locale, TimeZone timeZone) {
+    if (android.os.Build.VERSION.SDK_INT > Build.VERSION_CODES.FROYO) {
```

Comment:
This is always true. The minSdk is 8.

Reply:
Hey @JakeWharton, as the minSDK is 8 and the verification that I did is:
```
    if (android.os.Build.VERSION.SDK_INT > Build.VERSION_CODES.FROYO)
```
That means if the current API level is greater the FROYO ([API level 8](https://developer.android.com/reference/android/os/Build.VERSION_CODES.html#FROYO)), I will use `DateUtils` instead of `SimpleDateFormat`. To sum up, I will only use `SimpleDateFormat` when the API level is 8.

So, the verification can be false when the current API level is 8, right?

Reply:
yeah, that's right, formatDateRange with the timezone needs Gingerbread. (i also misread this because i normally see `if (SDK_INT >= GINGERBREAD)`, fwiw.)

Reply:
Let's just bump to 9 then. Froyo is dead.

Reply:
@NightlyNexus Good to know! I will use more this way to check. Thanks!
@JakeWharton I bumped to 9! 🎉 🎉 🎉 

I found an issue regarding the method `formatDateRange` that I was using it. It is not possible to set Locale/Timezone the way that I did. Then I fixed in this commit and I added a comment explaining better 😄 : https://github.com/square/android-times-square/pull/413/commits/53f24e5ec1df5d46fb3616be930401c32bb83eab